.github/workflows/do-spaces-workflow.yml

```diff
@@ -0,0 +1,44 @@
+name: Test and Deploy to DigitalOcean Spaces
+
+on: push
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Read .nvmrc
+      run: echo "##[set-output name=NVMRC;]$(cat .nvmrc)"
+      id: nvm
+
+    - name: Use Node.js (.nvmrc)
+      uses: actions/setup-node@v1
+      with:
+        node-version: "${{ steps.nvm.outputs.NVMRC }}"
+
+    - name: npm ci, test, and build
+      run: |
+        npm ci
+        npm test
+        npm run build
```

Comment:
Is this needed for the demo to build? Shouldn't this be `npm run demo:build` and the step below removed?

Reply:
`npm test` includes running `npm run build`, so I think this should be `npm run demo:build`?